docs/tutorials/upgrade-a-chain/scheduled-upgrade.md

```diff
@@ -52,7 +52,7 @@ dropped - that would require a storage migration, which is out of the scope of t
 Build the upgraded runtime.
 
 ```shell
-cargo build --release -p node-template-runtime
+cargo build -p node-template-runtime
```

Comment:
@joepetrowski want to get your confirmation. When we do a runtime upgrade, we want to compile the runtime in `release` mode, because the file size of the wasm in debugged mode is a lot bigger than the wasm in released mode. Is this true? I have this impression.